### DISCUSSION TITLE
How to test with dpeval

### DISCUSSION THREAD

**Discussion by EYErbil** [deepmodeling/deepmd-kit#4055] [Q&A]
Hello, I have 8000 single point frame multisystems of different structural layouts, I want to test those with the model I trained, but when I use : 
item = ["coord","box","energy","force"]
data = {}
for i in item:
    data[i] = np.load("../validation_data/Mo108S216H1_1/set.000/%s.npy"%i)

atype = np.loadtxt("../training_data/type.raw")
dp = DP("graph.pb")

pe, pf, pv = dp.eval(data["coord"][0], data["box"][0], atype)
#pe, pf, pv = dp.eval(data["coord"].reshape(1,-1,3), data["box"][0].reshape(1,3,3), atype)

print('orignal energy:%f'%data['energy'][0])
print('predicted energy:%f'%pe)

It only gives me a single test, I think I need to test every frame, how can I incorporate multisystems instead of systems.

**Comment by njzjz** [comment]
`dp.eval` accepts multiple frames:

```py
pe, pf, pv = dp.eval(data["coord"], data["box"], atype)
```
